apps/src/aichat/views/AichatView.tsx

```diff
@@ -3,38 +3,23 @@
 // This is a React client for an aichat level.
 
 import React from 'react';
-import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
-import Aichat from './Aichat';
-import {
-  sendSuccessReport,
-  navigateToNextLevel,
-} from '@cdo/apps/code-studio/progressRedux';
-import aichatLocale from '../locale';
-import styles from './aichat.module.scss';
 import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
+import ChatWorkspace from './ChatWorkspace';
+import moduleStyles from './aichat.module.scss';
 
 const AichatView: React.FunctionComponent = () => {
-  const dispatch = useAppDispatch();
-
-  const nextButtonPressed = () => {
-    const appType = 'aichat';
-    dispatch(sendSuccessReport(appType));
-    dispatch(navigateToNextLevel());
-  };
-
   return (
-    <div id="aichat">
-      <Aichat>
-        <PanelContainer id="instructions-panel" headerText="Instructions" />
-        <button
-          id="aichat-continue-button"
-          type="button"
-          onClick={() => nextButtonPressed()}
-          className={styles.buttonNext}
-        >
-          {aichatLocale.continue()}
-        </button>
-      </Aichat>
+    <div id="aichat" className={moduleStyles.aichat}>
+      {/* <div id="chat-instructions-area" className={moduleStyles.instructions}>
+        <PanelContainer id="chat-instructions-panel" headerText="Instructions">
+          <Instructions />
+        </PanelContainer>
+      </div> */}
```

Comment:
If code is commented out, there should be a comment explaining why, but it's probably better to remove it and then bring it back later if it becomes necessary.

Reply:
Will remove for now.